content/docs/command-reference/exp/init.md

```diff
@@ -64,71 +80,123 @@ $ dvc exp init './another_script.sh $MYENVVAR'
+  depends on can be found (if any). Overrides other configuration and default
+  value (`data/`).
 
-- `--live` - override the directory `path` for [DVCLive](/doc/dvclive), which
-  your experiment will write logs to. The default is `dvclive` directory, which
-  only comes to effect when used with `--type=dl`.
+- `--model` - set the path to the file or directory where the model(s) produced
+  by your experiment can be found (if any). Overrides other configuration and
+  default value (`models/`).
 
-- `--type` - selects the type of the stage to create. Currently it provides two
-  different kinds of stages: `default` and `dl`. If unspecified, `default` stage
-  is created.
+  > 💡 This could be used for any artifacts produced by your experiment.
 
-  `default` stage creates a stage with `metrics` and `plots` tracked by DVC
-  itself, and does not track live-created artifacts (unless explicitly
-  specified).
+- `--metrics` - set the path to the file or directory where the metrics produced
+  by your experiment can be found (if any). Overrides other configuration and
+  default value (`metrics.json`).
 
-  `dl` stage is intended for use in deep-learning scenarios, where metrics and
-  plots are tracked by [dvclive](/doc/dvclive) and supports tracking progress
-  while training a deep-learning model with
-  [checkpoints](/doc/command-reference/exp/run#checkpoints).
+- `--plots` - set the path to the file or directory where the plots produced by
+  your experiment can be found (if any). Overrides other configuration and
+  default value (`plots/`).
 
-- `-n <stage>`, `--name <stage>` - specify a custom name for the stage generated
-  by this command (e.g. `-n train`). The default is `train`.
+- `--live` - configure the `path` directory for [DVCLive](/doc/dvclive). This is
+  where experiment logs will be written. Overrides other configuration and
+  default value (`dvclive/`).
 
-  Note that the stage name can only contain letters, numbers, dash `-` and
-  underscore `_`.
+  > This only has an effect when used with `--type=dl`.
 
-- `-f`, `--force` - overwrite an existing stage in `dvc.yaml` file without
-  asking for confirmation.
+- `--explicit` - do not assume default locations of project dependencies and
+  outputs. You'll have to provide specific locations via other options or
+  `dvc config exp`. In `--interactive` this removes default values from prompts.
 
-- `--run` - runs the experiment after initializing it.
+- `-f`, `--force` - overwrite an existing stage in `dvc.yaml` file without
+  asking for confirmation (same as `dvc stage add --force`).
 
 - `-h`, `--help` - prints the usage/help message, and exit.
 
 - `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
   problems arise, otherwise 1.
 
 - `-v`, `--verbose` - displays detailed tracing information.
+
+## Example: interactive mode
+
+Let's prepare a raw data ingestion script to start running experiments on it.
+
+```dvc
```

Comment:
Why not a model training example? Data ingestion is a nice application to show how this can be useful even for non-training stages, but model training is the primary use case. It's unlikely for someone to run experiments on a data ingestion script.

Reply:
The data ingestion example produced a very small dvc.yaml file and then I had that note about using `stage add` to produce the same. Idk, I thought it was interesting. Anyway, I agree the training scenario is more applicable so 
I replaced all that now.